spec/package_spec.rb

```diff
@@ -55,25 +55,24 @@
   describe 'manifest_json' do
     it 'should return manifest json' do
       manifest = @package.manifest_json
-      expect(manifest[:name]).to eq('ABC')
-      expect(manifest[:author][:name]).to eq('John Smith')
-      expect(manifest[:author][:email]).to eq('john@example.com')
-      expect(manifest[:defaultLocale]).to eq('en')
-      expect(manifest[:private]).to eq(true)
-      expect(manifest[:location]).to eq('ticket_sidebar')
-      expect(manifest[:frameworkVersion]).to eq('0.5')
+      expect(manifest['name']).to eq('ABC')
+      expect(manifest['author']['name']).to eq('John Smith')
+      expect(manifest['author']['email']).to eq('john@example.com')
+      expect(manifest['defaultLocale']).to eq('en')
+      expect(manifest['private']).to eq(true)
+      expect(manifest['location']).to eq('ticket_sidebar')
+      expect(manifest['frameworkVersion']).to eq('0.5')
     end
   end
 
-  describe 'readified_js' do
+  describe 'compile_js' do
     it 'should generate js ready for installation' do
-      js = @package.readified_js(nil, 0, 'http://localhost:4567/')
+      js = @package.compile_js(app_id: 0, assets_dir: 'http://localhost:4567/0/')
 
       expected = <<HERE
-(function() {
-  with( ZendeskApps.AppScope.create() ) {
+with( ZendeskApps.AppScope.create() ) {
     require.modules = {
-        'a.js': function(exports, require, module) {
+        "a.js": function(exports, require, module) {
```

Comment:
it get's put there by does .to_json, so it will escape double quote.
`"my\"file"`